webrender/src/tiling.rs

```diff
@@ -489,9 +489,22 @@ impl AlphaRenderItem {
                         let key = AlphaBatchKey::new(batch_kind, flags, blend_mode, textures);
                         let batch = batch_list.get_suitable_batch(&key, item_bounding_rect);
 
+                        let mut cache_task_index = 0;
```

Comment:
this would look cleaner in functional style, `let cache_task_index = match prim_metadata.render_task {...};`